app/config/config_dev.yml

```diff
@@ -23,6 +23,16 @@ monolog:
 #        chromephp:
 #            type:  chromephp
 #            level: info
+        console:
+          type: console
+          process_psr_3_messages: false
+          channels: ['!event', '!doctrine', '!console']
+        # To follow logs in real time, execute the following command:
```

Comment:
I moved the comments to just underneath the `console` key. I think this way it is more clear that it concerns the console and it's more visible.